### PR TITLE
connect sentry performance

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

**Comment by KedziaPawel** [comment]
missing pods for ios, I know that you don't have ios but install pods on ubuntu and in the root execute the following commands:
> cd ios
> pod install

**Comment by mateuszpmroz** [comment]
> missing pods for ios, I know that you don't have ios but install pods on ubuntu and in the root execute the following commands:
> 
> > cd ios
> > pod install



@KedziaPawel 
I was trying to update the pod but I get error about missing `xcrun` and I didn't found a way to install it. It's possible that's it comming with `xcode` which I'm not able to install.

File: App.tsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ YellowBox.ignoreWarnings(['VirtualizedLists should never be nested inside', `\`-
 if (process.env.NODE_ENV !== 'development') {
   Sentry.init({
     dsn: 'https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289',
```
Comment (marek-siemieniuk-morawski) [inline App.tsx:21]: I think we shouldn't keep URLs like this in the codebase. Not sure how it solved so far but I'd recommend using an env variable here and configure Bitrise in the way to build the app with the env

**Review by marek-siemieniuk-morawski — CHANGES_REQUESTED** [review]

File: App.tsx (hunk 1 above)
Comment (KedziaPawel) [reply App.tsx:21]: yes, we have this sorted out on 3keys (develop), unfortunately, the 1key (master) doesn't have the possibility for configuring env as we added it later

**Comment by KedziaPawel** [comment]
doesn't work with react native

**Pull request closed without merging**